forge/forge/verify/verify.py

```diff
@@ -254,7 +257,80 @@ def verify_golden(
     device: "TTDevice",
     inputs: Tuple[Tensor],
     outputs: Tuple[torch.Tensor],
-    verify_cfg: VerifyConfig,
+    verify_cfg: DepricatedVerifyConfig,
 ):
 
     assert False  # Run ttnn golden
+
+
+def verify(
+    inputs: List[Union[torch.Tensor, tf.Tensor, tf.Variable]],
+    framework_model: Union[torch.nn.Module, tf.Module, tf.keras.Model],
+    compiled_model: CompiledModel,
+    verify_cfg: VerifyConfig = global_verify_config,
```

Comment:
Please, remove the `global_verify_config`.

There is no need for this you can just initialize with the default constructor. We should avoid any kind of global state, if possible...